tests/files.ts

```diff
@@ -35,6 +35,9 @@ test.serial(
 		const { body } = t.context;
 		state.set("fileId", body.id);
 		t.is(body.versions.length, 1, "File should have 1 version, the initial one");
+		unstableValues.add(body.id);
+		unstableValues.add(body.versions.map((x: any) => x.created_at));
```

Comment:
`Set#add` does not accept an array of values.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/add